packages/editor/src/components/editor-container.ts

```diff
@@ -45,8 +45,8 @@ export class EditorContainer extends WithDisposable(ShadowlessElement) {
     type: 'default',
   };
 
-  @state()
-  private showGrid = false;
+  @property()
+  showGrid = true;
```

Comment:
We can land this for now, but I think this can better be defined a property of config on the edgeless page block

Reply:
`showGrid` could be a configuration option for the user's workspace. Each user has their own configuration.